crates/nu-std/README.md

```diff
@@ -3,36 +3,78 @@
+│ 25 │ std log error               │ Log error message                                                   │
+│ 26 │ std log info                │ Log info message                                                    │
+│ 27 │ std log warning             │ Log warning message                                                 │
+│ 28 │ std path add                │ Add the given paths to the PATH.                                    │
+│ 29 │ std xaccess                 │ Get all xml entries matching simple xpath-inspired query            │
+│ 30 │ std xinsert                 │ Insert new entry to elements matching simple xpath-inspired query   │
+│ 31 │ std xtype                   │ Get type of an xml entry                                            │
+│ 32 │ std xupdate                 │ Update xml data entries matching simple xpath-inspired query        │
+├────┼─────────────────────────────┼─────────────────────────────────────────────────────────────────────┤
+│  # │            name             │                                usage                                │
+╰────┴─────────────────────────────┴─────────────────────────────────────────────────────────────────────╯
 
-## :toolbox: use the standard library in the REPL or in scripts
-in order to "import" the standard library to either the interactive [*REPL*][REPL] of
-`nushell` or inside some `.nu` script, you might want to use the
-[`use`](https://nushell.sh/commands/docs/use.html) command!
-```bash
-use std
 ```
 
-## :pencil2: contribute to the standard library
-- all the commands of the standard_library are located in [`std.nu`](std.nu)
-- the tests are located in files that have a name starting with "test_", e.g. [`test_std.nu`](test_std.nu)
-- a test runner, at [`tests.nu`](tests.nu), allows to run all the tests automatically
+## :toolbox: Using the standard library in the REPL or in scripts
+All commands in the standard library must be "imported" into the running environment 
+(the interactive read-execute-print-loop (REPL) or a `.nu` script) using the
+[`use`](https://nushell.sh/commands/docs/use.html) command.
 
-### :wrench: add new commands
-- add new standard commands by appending to [`std.nu`](std.nu)
-- add associated tests to [`test_std.nu`](tests_std.nu) or preferably to `test_<submodule>.nu`.
-    - define a new exported (!) `test_<feature>` command
-    - import the `assert` functions you need at the top of the functions, e.g. `use std.nu "assert eq"`
+You can choose to import the whole module, but then must refer to individual commands with a `std` prefix, e.g:
+```
+use std
+ . . .
+std log debug "Running now"
+std assert (1 == 2)
+```
 
-### :test_tube: run the tests
-the following call should return no errors
-```bash
-NU_LOG_LEVEL=DEBUG cargo run -- crates/nu-std/tests.nu
+Or you can enumerate the specific commands you want to import and invoke them without the `std` prefix.
 ```
+use std log assert
+. . .
+log debug "Running again"
+assert (2 == 1)
```

Comment:
with the current module system, that would not work :eyes: 
```bash
> use std [log assert]
Error: nu::parser::export_not_found

  × Export not found.
   ╭─[entry #2:1:1]
 1 │ use std [log assert]
   ·          ─┬─
   ·           ╰── could not find imports
   ╰────
```

however, applying the following would fix it
```diff
-use std log assert
+use std ['log debug' assert]
```

Reply:
I definitely should have verified the example works as documented.  I'll update this.
But why does `use` require second argument to be an array? Doesn't `use` take any number of arguments (rather than exactly 2, the second of which is an array ...)?

Reply:
> I definitely should have verified the example works as documented. I'll update this.
>
no worries, this is why humanity invented the Review :wink: 

> But why does `use` require second argument to be an array? Doesn't `use` take any number of arguments (rather than exactly 2, the second of which is an array ...)?
>
i do not know the "why" but...
```bash
>_ use std 'log debug' assert
Error: nu::parser::extra_positional

  × Extra positional argument.
   ╭─[entry #1:1:1]
 1 │ use std 'log debug' assert
   ·                     ───┬──
   ·                        ╰── extra positional argument
   ╰────
  help: Usage: use <module> (members)
```
according to `help use`, it takes a `members` but not a `...members` :open_mouth:

Reply:
:+1: